docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+version: '3.2'
+services:
+  curity-service:
+    image: curity/idsvr
+    ports:
+      - 6749:6749
+      - 8443:8443
+    environment:
+      - ADMIN=true
+      - SERVICE_ROLE=default
+      - LICENSE_KEY=${LICENSE_KEY:?err}
```

Comment:
these have to be set manually? From the README above:

> edit the `docker-compose.yaml` file and configure the variables `ADMIN_PASSWORD` and `LICENSE_KEY`

They can't be set in the environment when `docker-compose` is run?

Reply:
Yeah they can be environment variables

Reply:
Can you add an example to the docs on how to pass them in that way? I tried `ADMIN_PASSWORD=foo LICENCE_KEY=bar make test` and `make test ADMIN_PASSWORD=foo LICENCE_KEY=bar` and `export ADMIN_PASSWORD=foo LICENCE_KEY=bar ; make test`. None of those seemed to work for me.

Reply:
All of those ways work for me, you can also add the values in a .env file. I ll add a note in the comments